.tours/3---dynamic flows.tour

```diff
@@ -0,0 +1,70 @@
+{
+    "title": "3 - Dynamic Flows",
+    "steps": [
+        {
+            "file": "es-space-mgmt/main/default/aura/reservationHelperAura/reservationHelperAura.cmp",
+            "description": "The reservationHelper Aura component render flow interviews dynamically, by using the lightning:flow base component",
+            "line": 17,
+            "selection": {
+                "start": {
+                    "line": 17,
+                    "character": 1
+                },
+                "end": {
+                    "line": 20,
+                    "character": 1
+                }
+            }
+        },
+        {
+            "file": "es-space-mgmt/main/default/aura/reservationHelperAura/reservationHelperAuraHelper.js",
+            "description": "Use the startFlow method to start the create reservation flow",
+            "line": 14
+        },
+        {
+            "file": "es-space-mgmt/main/default/aura/reservationHelperAura/reservationHelperAura.cmp",
+            "description": "The reservationHelper Aura component embeds lwc component reservationHelper. reservationHelper lwc component acts a bridge between reservationHelperForm lwc flow screen component and reservationHelperAura component. The component uses LMS to subscribe to flow status change event and customer tile selection event",
```

Comment:
This needs to be split into multiple steps. It is difficult to follow and remember the names of the components.. May be navigate to each component when speaking about it.

Reply:
I have broken it into multiple steps. The reality is this is going to be an issue until we get rid of the aura component. Maybe once LWC gives this feature it will get better!